lib/components/Article.js

```diff
@@ -29,8 +29,8 @@ const styles = {
 const dateDisplay = (dateString) => new Date(dateString).toDateString();
 
 // {} these are used to access inner property of context object
-const Article = (props, { store }) => {
-	const { article } = props;
+const Article = (props) => {
+	const { article, store } = props;
 	const author = store.lookupAuthor(article.authorId);
 	return (

```

Comment:
Make sure testData is getting passed by console.log

Reply:
it was actually failing tests